### DISCUSSION TITLE
Udp protocol support between api gateway envoy and microservice intranamespace

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#547] [Q&A] [answered]
### What product are you having troubles with?

Marketplace

### What Console version are you using?

13.x

### Description

Hi!
The api gateway envoy does support connectivity using the UDP protocol? 
If so, do you have some example of how to configure the envoy gateway?

In my scenario I have a microservice that accepts requests in one or more UDP ports and I want to make sure that the envoy gw is ready to handle the requests

**Comment by akelity** [comment, marked as answer]
Thank you @elmalakomar, for the question and for contributing to the documentation page.

In this Mia-Platform Docs more details on how to address this topic: https://docs.mia-platform.eu/docs/development_suite/api-console/advanced-section/api-gateway-envoy/udp
